src/frontend/build.gradle.kts

```diff
@@ -29,7 +29,7 @@ tasks {
     }
     named<NpmTask>("npm_run_buildAdmin") {
         outputs.cacheIf { true }
-        environment.set(mapOf("PUBLIC_URL" to "/admin"))
+        environment.set(mapOf("PUBLIC_URL" to "/admin", "BUILD_PATH" to "build", "REACT_APP_MODE" to "main"))
```

Comment:
I'm not sure if it is fine to have same build for both admin and overlay. Wouldn't they override each other?